docs/guides/node-operator-manual.md

```diff
@@ -276,3 +326,43 @@ docker run --rm -it \
   --datadir /root/.lighthouse/data \
   --directory /root/validator_keys
 ```
+
+### Checking the keys of all Lido Node Operators
+
+After keys are submitted, it's critical to first wait for the key submission transactions to be mined and after that check all Lido keys for validity.
+
+Never vote for increasing the key limits of Node Operators before verifying new keys are present and valid.
```

Comment:
Please, check there is enough valid keys submitted before voting for key limit increase.